src/logUtils.ts

```diff
@@ -55,7 +53,8 @@ export async function handleLogViaKamelCli(integrationName: string) : Promise<vo
 			proc.stdout.on('data', async (data: string) => {
 				if (data.length > 0) {
 					const buf = Buffer.from(data);
-					const text = stripAnsi(buf.toString());
+					const stripAnsi = await import('strip-ansi');
```

Comment:
if we can finda way to avoid import statement in teh middle of the code that would be nice

Reply:
Yes, I would also prefer to have it done in a nice way.

Reply:
created a Jira for that... https://issues.redhat.com/browse/FUSETOOLS2-1420